### ISSUE TITLE
Postman Pre-request Script Hangs

### ISSUE THREAD

**Issue by kwhat** [a85/POSTMan-Chrome-Extension#3730]
## App Details:
```
Postman for Mac
Version 5.3.2 (5.3.2)
OS X 10.11.6 (15G17023) / x64
```

## Issue Report:
 1. Encountered this recently
 2. Expected Pre-request Script to execute.
 3. Console logs: ```Cannot read property 'headers' of undefined
(anonymous) @ console.js:153296
wrapper @ common.js:5658
handleInternalMessage @ console.js:152199
handleElectronMessage @ console.js:152231
(anonymous) @ console.js:152170
emitTwo @ events.js:106
emit @ events.js:191```
 4. N/A

## Steps to reproduce the problem
This only happens with 1 endpoint in particular.  I create a new endpoint, point it to my destination.  Add an argument, click send, request goes through, no issues.  If I add any javascript to Pre-request Scripts, and I do mean **ANY** javascript `var a = 1;` or even `/**/`, the request hangs... and when I cancel I get the error from the console.

Other endpoints work as expected.  If i remove the endpoint and recreate it, same issue.  Noting shows up in the postman console until I cancel and then it just has an ambiguous "ERROR null."

**Comment by SamvelRaja** [comment]
@kwhat  We are having a hard time in reproducing the issue you mentioned, can you provide us a screencast of the issue you are facing. If the data is much sensitive to show in public, can you send us a mail to help[at]getpostman[dot]com with your registered email id?

**Comment by kwhat** [comment]
Hi @SamvelRaja,

I have no idea what was going on with the endpoint.  I basically copy pasted all of the values from the broken endpoint to a new one and it just worked.  Duplicating the misbehaving endpoint, duplicated the error.  I ended up just deleting the endpoint and recreated it.  I did notice that the exported item had a strange anomaly in the request body, despite setting it to post 'form-data' and the ui reflecting that with variables saved in the body, it shows as mode: "raw", "raw": "".  I have sent an email to the provided address.
